src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Microsoft.VisualStudio.ProjectSystem.Managed.VS.csproj

```diff
@@ -16,6 +16,7 @@
     <NoWarn>$(NoWarn);NU5125</NoWarn>    
   </PropertyGroup>
   <ItemGroup>
+    <ProjectReference Include="..\LogProviders\Microsoft.VisualStudio.ProjectSystemTools.BackEnd.csproj" />
```

Comment:
Why does this need to be referenced here?

Reply:
I forgot to remove it, thanks for catching that!